src/OrchardCore/OrchardCore.Email.Abstractions/SmtpDeliveryMethod.cs

```diff
@@ -1,5 +1,6 @@
 namespace OrchardCore.Email
 {
+    // TODO: The enum has be moved to OC.Email.Smtp, so remove it on the upcoming major release.
```

Comment:
"has been" or "has to be"? Also check the other TODO comments, please.
BTW is there a procedure to make sure that is done before the next major release?

Reply:
We might move this now and mention it in the release notes as we are doing nowadays